### DISCUSSION TITLE
Increasing/decreasing speed using hotkey in mods menu

### DISCUSSION THREAD

**Discussion by cirru05** [ppy/osu#21444] [Ideas]
In the current state of the game you have to either select the Double Time or the Nightcore (or the Half Time) mod and then go into the Mod Customisation menu after which you still have to slide the slider in order to change a beatmap's speed to something else than x0.75 or x1.5.
It's not too much of a job to go through this, but I think it would be even more convenient if players could just increase/decrease the speed of the music while on the mods menu without having to click on anything (just like the shortcuts to other mods, like F for DT and G for NC). In my opinion setting one hotkey on increasing and one on decreasing the speed by... like x0.05 or x0.1 at a time would save quite some time for players who change how rapidly they want to play their maps regularly.

**Comment by smoogipoo** [comment]
For sure. I think this would be useful but getting it to work with mods is an interesting task - probably a special case specifically for the DT mod.
